pandas/io/tests/test_parsers.py

```diff
@@ -6,39 +6,35 @@
 from datetime import datetime
 import csv
 import os
-import sys
-import re
-import nose
 import platform
-
+import re
+import sys
+from datetime import datetime
 from multiprocessing.pool import ThreadPool
 
-from numpy import nan
+import nose
 import numpy as np
-from pandas.io.common import DtypeWarning
+import pandas.lib as lib
+import pandas.parser
+from numpy import nan
+from numpy.testing.decorators import slow
+from pandas.lib import Timestamp
 
+import pandas as pd
+import pandas.io.parsers as parsers
+import pandas.tseries.tools as tools
+import pandas.util.testing as tm
 from pandas import DataFrame, Series, Index, MultiIndex, DatetimeIndex
+from pandas import compat
 from pandas.compat import(
     StringIO, BytesIO, PY3, range, long, lrange, lmap, u
 )
+from pandas.compat import parse_date
+from pandas.io.common import DtypeWarning
 from pandas.io.common import URLError
-import pandas.io.parsers as parsers
 from pandas.io.parsers import (read_csv, read_table, read_fwf,
                                TextFileReader, TextParser)
-
-import pandas.util.testing as tm
-import pandas as pd
-
-from pandas.compat import parse_date
-import pandas.lib as lib
-from pandas import compat
```

Comment:
Just reorganized the imports to be legible. Notice how now we can see a bit of simplicity that can be made after the organizations:
```
from pandas import DataFrame, Series, Index, MultiIndex, DatetimeIndex
from pandas import compat
from pandas.compat import(
    StringIO, BytesIO, PY3, range, long, lrange, lmap, u
)
from pandas.compat import parse_date
from pandas.io.common import DtypeWarning
from pandas.io.common import URLError
```